backend/src/main/java/com/google/sps/servlets/SearchMatchServlet.java

```diff
@@ -116,10 +123,15 @@ private boolean isExpired(Participant participant) {
   }
 
   /** Send JSON response for expired participant that has been removed from datastore */
-  private void sendExpiredResponse(HttpServletResponse response) throws IOException {
+  private void sendExpiredResponse(HttpServletResponse response, Participant participant)
+      throws IOException {
     JSONObject expired = new JSONObject();
     expired.put(JSON_MATCH_STATUS, "expired");
-
+    try {
+      emailNotifier.sendExpiredEmail(getUsername(), userService.getCurrentUser().getEmail());
+    } catch (MessagingException e) {
+      e.printStackTrace();
+    }
```

Comment:
Ok so now if there's a problem constructing an email notifier here, how should the servlet behave? Is it okay to continue on without sending an email? @jzwang8 @dhruvi618

Reply:
If there's a problem constructing an email notifier here, I think it would okay to continue on without sending an email if the servlet could still return the JSON object to the frontend so we can notify the user that way